test/Interop/Cxx/stdlib/print-libcxx-module-interface.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=CxxStdlib -source-filename=x -enable-experimental-cxx-interop -enable-objc-interop -module-print-submodules | %FileCheck %s
```

Comment:
Is this test different from `test/Interop/Cxx/stdlib/libcxx-module-interface.swift`?